AvanceJuanManuelGarcia.cpp

```diff
@@ -0,0 +1,260 @@
+#include <iostream>
+#include <string>
+using namespace std;
+
+
+class Libros
+{
+private:
+	int id;
+	string titulo;
+	int cantidadEnStock;
+	string disponibleParaKit;  //si o no
```

Comment:
Para variables que solo almacena dos valores  "si" o "no".
Recomendable utilizar booleanos
```suggestion
	bool estaDisponibleParaKit;  //si o no
```